src/xcode/fastlane/Fastfile

```diff
@@ -63,13 +65,40 @@ platform :ios do
   desc "Lint code"
   lane :lint do
     swiftlint(
+      executable: "../../vendor/swiftlint",
       mode: :lint,
-      output_file: "swiftlint.result.json",
+      output_file: "swiftlint.html",
       config_file: "ENA/.swiftlint.yml",
       strict: true,
+      quiet: true,
+      no_cache: false,
+      reporter: "html"
+    )
+  end
+
+  desc "Static code analysis via swiftlint's experimental feature"
+  lane :analyze do
+    UI.important "This lane uses experimental features and takes very long to complete!"
+    UI.message "Better get a coffee… ☕️"
+
+    build_ios_app(
+      skip_codesigning: true,
+      export_method: "ad-hoc",
+      skip_package_ipa: true,
+      #destination: "platform=iOS Simulator,OS=14.4,name=iPhone 11",
+      scheme: "ENA",
+      buildlog_path: "DerivedData/logs/"
+    )
+    swiftlint(
+      executable: "../../vendor/swiftlint",
+      mode: :analyze,
+      compiler_log_path: "DerivedData/logs/ENA-ENA.log",
+      output_file: "swiftlint.analyze",
+      config_file: "ENA/.swiftlint.yml",
+      strict: false,
       quiet: false,
-      raise_if_swiftlint_error: true,
```

Comment:
does this mean a linter error does not raise a failure?

Reply:
I think I missed this when I was working on a better lint output…

Reply:
@pascal-brause I tested the current setup and it fails fine on linter errors: https://app.circleci.com/pipelines/github/corona-warn-app/cwa-app-ios/17289/workflows/e80b657e-aed2-4255-b401-22e839be3c55/jobs/57223